prober/derp.go

```diff
@@ -348,6 +348,8 @@ func newConn(ctx context.Context, dm *tailcfg.DERPMap, n *tailcfg.DERPNode) (*de
 	priv := key.NewNode()
 	dc := derphttp.NewRegionClient(priv, l, nil /* no netMon */, func() *tailcfg.DERPRegion {
 		rid := n.RegionID
+		// Allow the prober to monitor nodes marked as STUN only in the default map
+		n.STUNOnly = false
```

Comment:
Do the stun only derps all respond to disco?

If they don't this may raise some alerts for probes failing with meshing.

Reply:
Right now this code only runs on things in the superset derpmap, which only has one server marked STUN only. Other derps are monitored with different code.

Reply:
👍🏼